tests/test_isort.py

```diff
@@ -84,6 +72,32 @@ def test_precommit(self, _ok_mock):
     @patch("autohooks.plugins.isort.isort.error")
     @patch("autohooks.plugins.isort.isort.get_staged_status")
     def test_precommit_staged(self, staged_mock, _error_mock, _ok_mock):
-        staged_mock.return_value = [StatusEntry("M  tests/isort_test.py")]
-        ret = precommit()
-        self.assertFalse(ret)
+        CONTENT: str = (  # noqa: I001
+            """
+# pylint: disable-all
+from io import StringIO, BytesIO, FileIO  # pylint: disable=unused-import
+import sys
+import black
+
+import autohooks
+
+cmd = ["pylint", "autohooks/pluginrecommit_stages/pylint/pylint.py"]
+import subprocess  # pylint: disable=
+
+# status = subprocess.call(cmd)
+iofile = "tmp.txt"
+# status = subprocess.call(cmd, stdout=iofile)
+# blah blah lots of code ...
+
+status = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+out, err = status.communicate()
+print(out.decode(encoding="utf-8"))
+print(err.decode(encoding="utf-8"))
+"""
```

Comment:
```suggestion
            """from io import StringIO, BytesIO, FileIO
import sys
import black

import autohooks

cmd = ["pylint", "autohooks/pluginrecommit_stages/pylint/pylint.py"]
import subprocess  
"""
```

Actually we can remove all this ...